### ISSUE TITLE
Ability to specify separate eslint configuration file for Save fixes.

### ISSUE THREAD

**Issue by SiarheiKuchukBelitsoft** [microsoft/vscode-eslint#931]
Hello

I have an idea. It would be awesome to have a possibility to have 2 eslint configuration files. One for checks etc (normal one) and one for Save fixes rules.

Thanks

**Comment by dbaeumer** [comment]
This got discussed here: https://github.com/microsoft/vscode-eslint/issues/902#issuecomment-588805954

Objection to mark it as a duplicate.

**Comment by SiarheiKuchukBelitsoft** [comment]
Hi

Just in case

@dbaeumer 
Scenario in which i'm using 2 set of configs: 1 config for check and 1 config for autofix on save. I want only safe rules to be applied to Save, because after some rules applying unsafe changes are performed and there's no way to prevent it (turn rule off is not an option).

I've implemented saving with execution of fixing eslint through node with different config via plugin Run on Save. But on my development machine it takes 3 seconds (!!!!) to wait till file is updated on my hi-end development PC.

And moreover if i continue edit, VS Code detects that file was changed and asks me to merge them both files.

It is really needed to have that in Ms Eslint plugin, because it will do it fast.
